AlfrescoApp/Utils/Categories/AlfrescoNode+Sync.h

```diff
@@ -16,7 +16,8 @@
  *  limitations under the License.
  ******************************************************************************/
 
-#import "AlfrescoNode.h"
+#import <AlfrescoSDK-iOS/AlfrescoSDK.h>
```

Comment:
You are right, it is not needed as I just performed additional checks. Will remove the import.